libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json

```diff
@@ -312,6 +312,11 @@
         "comment": "Message of the scheduled email.",
         "limit": 0
     },
+    "dialogs.schedule.email.unsubscribed.recipients": {
+        "value": "{n, plural, one {1 recipient unsubscribed from this schedule} other {# recipients unsubscribed from this schedule}}.",
+        "comment": "Translate only uppercase in '{n, plural, one {1 RECIPIENT UNSUBSCRIBED FROM THIS SCHEDULE} other {# RECIPIENTS UNSUBSCRIBED FROM THIS SCHEDULE}}.'",
```

Comment:
This kind of comments is misleading, I think. I know we use it elsewhere. I would say just "translate with correct plural form for given language"